compiler/passes/codegen.cpp

```diff
@@ -625,6 +625,20 @@ static void codegen_header_compilation_config() {
     genComment("Filename Lookup Table");
     genFilenameTable();
 
+    //This adds the Chapel symbol table to the file
```

Comment:
I think it would be useful to have a comment near this code indicating what the format of chpl_funSymTable is. (a 1-D array with 4 elements per entry: cname, Chapel name, Chapel file name, Chapel line number).